webapp/channels/src/components/file_preview_modal/image_preview.tsx

```diff
@@ -36,13 +37,41 @@ export default function ImagePreview({fileInfo, canDownloadFiles}: Props) {
             className='image_preview'
             href='#'
         >
-            <img
-                className='image_preview__image'
-                loading='lazy'
-                data-testid='imagePreview'
-                alt={'preview url image'}
-                src={previewUrl}
-            />
+            <TransformWrapper>
+                {({zoomIn, zoomOut, resetTransform}) => (
+                    <>
+                        <div className='image_preview_zoom_actions__actions'>
+                            <button
+                                onClick={() => zoomIn()}
```

Comment:
For these click handlers, you can actually just pass them in directly as `onClick={zoomIn}` without creating a new function. It mostly just simplifies the code a bit, but I'd assume there's some miniscule performance benefit from not allocating a new function every time this updates

Reply:
There is a type conflict when you pass in the function directly. Maybe there is a way to make the types cooperate? But this was the fix that copilot suggested.

Reply:
Ah, I see. `zoomIn` takes its own parameters and passing it directly tries to pass the click event as the first one of those. These are fine then